scripts/flattener.py

```diff
@@ -822,6 +879,40 @@ def drop_cols(celltype_col):
 			cxg_obs.drop(columns=column_drop, inplace=True)
 
 
+# Add ontology term names to CXG standardized columns
+def add_labels():
+	global cxg_adata
+	global cxg_adata_raw
+	query_url = urljoin(server, 'search/?type=OntologyTerm&field=term_id&field=term_name&limit=all')
+	r = requests.get(query_url, auth=connection.auth)
+	try:
+		r.raise_for_status()
+	except requests.HTTPError:
+		sys.exit("Error in ontology term ids and names: {}".format(query_url))
+	else:
+		ontology_df = pd.DataFrame(r.json()['@graph'])
+	id_cols = ['assay_ontology_term_id','disease_ontology_term_id','cell_type_ontology_term_id','development_stage_ontology_term_id','sex_ontology_term_id',\
+			'tissue_ontology_term_id','organism_ontology_term_id','self_reported_ethnicity_ontology_term_id']
+	for col in id_cols:
+		name_col = col.replace("_ontology_term_id","")
+		cxg_adata.obs[name_col] = cxg_adata.obs[col]
+		for term_id in cxg_adata.obs[name_col].unique():
+			if term_id == 'PATO:0000461':
+				term_name = 'normal'
+			elif term_id == 'PATO:0000384':
+				term_name = 'male'
```

Comment:
Is female/PATO:0000383 missing?

Reply:
@jahilton Thanks! I've pushed a new commit.